CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Please enumerate **all user-facing** changes using format `<githib issue/pr numb
 
 ## 0.5.0
 
+* [#1049](https://github.com/kroxylicious/kroxylicious/pull/1049): Add deprecated EnvelopeEncryption filter to ease migration to RecordEncryption filter
```

Comment:
there's also a block in the Changes, deprecations, removals that should be update to talk about the deprecation/replacement rather than rename.

Reply:
Good point I had meant to update that

Reply:
I think rename is the still the right description, as we aren't changing the API/functionality but I have updated it to clarify the old name still works.